models/Issue.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var mongoose = require('mongoose');
+
+var issueSchema = mongoose.schema({
+	author_id: Number,
+	title: String,
+	content: String,
+	votes: {
+		up: Number,
+		down: Number
```

Comment:
Is it possible to add a field that is total_votes? This would create an easier way to sort the issues later on, if not we can do it when we sort, but it might make it easier to add the field here.